### PR TITLE
POL-113 Azure Unused SQL Databases

### PR DESCRIPTION
**Pull request by nshivappa**
Initial release

### Description

Policy template checks for Azure SQL Databases that are unused by reviewing the DB connections and delete them after user approval.

### Issues Resolved

This policy gets a list of Azure SQL Databases and uses the DB Connection metric to check for successful connections over a 30-day period. If there are no successful DB Connections the policy will terminate the SQL databases after the user approval.

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### Template running link

https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5daff371a075f60089bf53a2

### Screenshot

image

### CODE REVIEW

File: cost/azure/unused_sql_databases/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Azure Unused SQL Databases
+ 
+### What it does
+
+This Policy template checks for Azure SQL Databases that are unused by reviewing the DB connections and delete them after user approval.
+
+### Functional Details
+
+- This policy gets a list of Azure SQL Databases and uses the DB Connection metric to check for successful connections over a 30-day period. If there are no successful DB Connections the policy will terminate the SQL databases after the user approval.
```
Comment (cdwilhelm) [inline cost/azure/unused_sql_databases/README.md]: The bullet isn't needed here.

File: cost/azure/unused_sql_databases/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+1. Follow steps to [Create an Azure Active Directory Application](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application)
+1. Grant the Azure AD Application access to the necessary subscription(s)
+1. [Retrieve the Application ID & Authentication Key](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-application-id-and-authentication-key)
+1. Create RightScale Credentials with values that match the Application ID (Credential name: `AZURE_APPLICATION_ID`) & Authentication Key (Credential name: `AZURE_APPLICATION_KEY`)
+1. [Retrieve your Tenant ID](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#get-tenant-id)
+
+### Actions
+  
+- Sends an email notification
+- delete unused SQL Databases after approval
```
Comment (cdwilhelm) [inline cost/azure/unused_sql_databases/README.md]: Capitalize first letter for Delete

File: cost/azure/unused_sql_databases/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+v1.0
+-----
```
Comment (cdwilhelm) [inline cost/azure/unused_sql_databases/CHANGELOG.md]: Add CHANGELOG header.

**Comment by cdwilhelm** [comment]
@nshivappa  Please check the conflicting file also.

File: cost/azure/unused_sql_databases/CHANGELOG.md (hunk 3 above)
Comment (nshivappa) [reply cost/azure/unused_sql_databases/CHANGELOG.md]: Added, thanks.

**Pull request merged**